src/ansys/geometry/core/tools/problem_areas.py

```diff
@@ -282,16 +302,19 @@ class SplitEdgeProblemAreas(ProblemArea):
     ----------
     id : str
         Server-defined ID for the body.
-    grpc_client : GrpcClient
-        Active supporting geometry service instance for design modeling.
     edges : List[str]
         List of edges associated with the design.
+    grpc_client : GrpcClient
+        Active supporting geometry service instance for design modeling.
```

Comment:
```suggestion
        Active supporting Geometry Service instance for design modeling.
```

Geometry Service should be capitalized. Please correct everywhere